From b40a6a958b05a17cc919eca4a1a24c4273b9b38e Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Fri, 26 Feb 2016 11:36:46 +0800
Subject: [PATCH 0959/5242] MLK-12456 ARM: dts : add spi1 IOMUX sleep state

commit  14900b4c51187a6534f3eb672bd7a6400bb097fb from
https://source.codeaurora.org/external/imx/linux-imx.git

Add spi1 IOMUX sleep state in imx7d-12x12-lpddr3-arm2.dts.

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index f991a00..421c29f 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -157,8 +157,9 @@
 &ecspi1 {
 	fsl,spi-num-chipselects = <1>;
 	cs-gpios = <&gpio4 19 0>;
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_ecspi1_1 &pinctrl_ecspi1_cs_1>;
+	pinctrl-1 = <&pinctrl_ecspi1_1 &pinctrl_ecspi1_cs_1>;
 	status = "disabled";
 
 	spi_flash1: m25p80@0 {
-- 
1.7.9.5

